src/components/ConfigMapSecret/ConfigMapSecret.components.tsx

```diff
@@ -246,13 +242,13 @@ export function ConfigMapSecretContainer({
             }
             return history.push(getURL('create'))
         } else {
+            getData()
```

Comment:
This should only be in the else case. why do we need it at the time of closing?

Reply:
fixed that